docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -667,6 +713,451 @@ Script.serveFunction("CSK_ModuleName.setSelectedInstance", setSelectedInstance)
 
 Besides, there are also functions to dynamically add/reset the number of instances.
 
+### FlowConfig Functions and Events
+
+If a module should be available within the FlowConfig, there are some events and function it should provide.
+
+Additionally, there are some other relevant lines of code already mentioned in other chapters. Please search for 'FlowConfig' on this documentation page to find other relevant lines of code.
+
+**INFO: Before you try to extend a module with this feature, you should be sure to understand all the other CSK basics!**
```

Comment:
OK, but alternatively to me "other" sounds strange, Optional suggestion: 
"understand all the basics of CSK"

Reply:
Done